types/test.ts

```diff
@@ -0,0 +1,13 @@
+import remark = require('remark')
+import hastscript = require('hastscript')
+import remarkAutolinkHeading from 'remark-autolink-heading'
+
+remark().use(remarkAutolinkHeading)
+remark().use(remarkAutolinkHeading, {behavior: 'wrap'})
```

Comment:
For good measure, it may help to have an expected error case as well
```suggestion
remark().use(remarkAutolinkHeading, {behavior: 'wrap'})
// $ExpectError
remark().use(remarkAutolinkHeading, {notAnOption: true})
```